modules/planning/conf/planning_config.pb.txt

```diff
@@ -75,7 +75,7 @@ default_task_config: {
     high_speed_threshold: 12.58
     low_speed_threshold: 7.5
     minimal_kappa: 0.00001
-    point_extension: 1.0
+    point_extension: 0.0
```

Comment:
When the changes are finalized, remove the parameters in the protobuf.

Reply:
sure